essi-nginx.nginx

```diff
@@ -25,7 +25,7 @@ server {
     proxy_set_header        Connection $connection_upgrade;
 
     # Fix the "It appears that your reverse proxy set up is broken" error.
-    proxy_pass          http://essi_app;
+    proxy_pass          http://essi;
```

Comment:
This looks like a hostname, but it is actually referring to the upstream defined above.
```suggestion
    proxy_pass          http://essi_app;
```